packages/cli-platform-android/src/config/__fixtures__/files/AndroidManifest-few-activities.xml

```diff
@@ -0,0 +1,20 @@
+<manifest
+  xmlns:android="http://schemas.android.com/apk/res/android">
+  <application android:name=".ExampleApplication">
+    <activity android:name=".ExampleAppActivity">
```

Comment:
commenting here but valid for the whole PR:

Activities can also be specified with the fully qualified package name IIRC like `android:name="com.example.ExampleAppActivity"`.

Source: https://developer.android.com/guide/topics/manifest/activity-element#nm

Can we add examples for it in the tests?